goldstone/core/models.py

```diff
@@ -2634,3 +2639,87 @@ def integration(cls):
     def resourcetype(cls):
 
         return "routers"
+
+
+class SavedSearch(models.Model):
+    """Defined searches, both system and user-created."""
+
+    # This object's Goldstone UUID.
+    uuid = UUIDField(version=1, auto=True, primary_key=True)
+
+    name = models.CharField(max_length=64)
+
+    owner = models.CharField(max_length=64)
+
+    description = models.CharField(max_length=1024, default='Defined Search')
+
+    query = models.CharField(max_length=512,
```

Comment:
Based on what I'm seeing in the compliance testing, the query field needs to handle much longer strings.  Like three times longer, easily.

I strongly suggest changing this to a TextField.